### PR TITLE
Update file for backward compatibility

### PR DESCRIPTION
**Pull request by quintonn**
Updated hook to be backward compatible with older version of nodejs

### CODE REVIEW

**Comment by alexandergoncharov** [comment]
Hi @quintonn,

Thanks for contributing!

File: hooks/afterPluginAdd.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = function (ctx) {
+
     if (ctx.cmdLine.includes("cordova-plugin-code-push")) {
```
Comment (alexandergoncharov) [inline hooks/afterPluginAdd.js]: Please replace this `includes()` method with `indexOf()`
Let's get rid of all `includes()` here for backword compatibility

**Comment by quintonn** [comment]
Hi @alexandergoncharov,
I've replaced the other includes with an indexOf also.

**Comment by quintonn** [comment]
@alexandergoncharov 
Done

**Pull request merged**